src/Microsoft.IdentityModel.Logging/IdentityModelEventSource.cs

```diff
@@ -112,5 +141,17 @@ public static EventLevel LogLevel
                 _logLevel = value;
             }
         }
+
+        private string PrepareMessage(string message, EventLevel level)
+        {
```

Comment:
This is a private method and will be called from one of the Write methods variants which are tied to a specific EventLevel. The 'level' will never be out of range.